payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AuBankAccountNumberConfig.kt

```diff
@@ -41,17 +41,22 @@ class AuBankAccountNumberConfig : TextFieldConfig {
             return TextFieldStateConstants.Error.Blank
         }
 
-        if (input.length < LENGTH) {
+        if (input.length < MINIMUM_LENGTH) {
             return TextFieldStateConstants.Error.Incomplete(
                 StripeR.string.stripe_becs_widget_account_number_incomplete
             )
         }
 
+        if (input.length < MAXIMUM_LENGTH) {
+            return TextFieldStateConstants.Valid.Limitless
+        }
+
         return TextFieldStateConstants.Valid.Full
     }
 
     private companion object {
-        const val LENGTH = 9
+        const val MINIMUM_LENGTH = 4
```

Comment:
Ideally, we would only enable a minimum length of 4 for `Cuscal` but given that LUXE provides these fields separately and we have no way of currently observing other form field values, I haven't been able to think of a solution that either:
- Does make major changes to element system to observe values coming from other elements.
- Make assumptions on an LPM's elements (ignore LUXE and make a custom `Element` to handle all of Becs.